Nos/Views/Home/HomeFeedView.swift

```diff
@@ -63,10 +62,7 @@ struct HomeFeedView: View {
         ZStack {
             let homeFeedFilter = Filter(
                 authorKeys: user.followedKeys, 
-                kinds: [.text, .delete, .repost, .longFormContent, .report], 
-                limit: 100, 
-                since: nil,
-                keepSubscriptionOpen: true
+                kinds: [.text, .delete, .repost, .longFormContent, .report]
```

Comment:
I think it's fine that this filter has keepSubscriptionOpen set to false because it is used for paging. However, this isn't immediately apparent without diving into the code and discovering that the filter is used in PagedNoteDataSource.

Ideally, the `Filter` object shouldn't be responsible for this decision, but refactoring might not be necessary at this moment. As a compensation, we should manually set `keepSubscriptionOpen` to false inside the code of `PagedNoteDataSource` to ensure that `Filter` aligns with the `PagedNoteDataSource` requirements and to clearly communicate this to developers reading the code.

Reply:
I think you're right. Until recently the code creating the filter and the code opening a subscription with the RelayService were the same. But with `PagedNoteDataSource` that is no longer true, and the decision whether to keep the subscription open really is up to the code opening the subscription. But I'll punt on that refactor to a later time.

> As a compensation, we should manually set keepSubscriptionOpen to false inside the code of PagedNoteDataSource to ensure that Filter aligns with the PagedNoteDataSource requirements and to clearly communicate this to developers reading the code.

This is already happening, it's just in `PagedRelaySubscription` not `PagedNoteDataSource`. The `PagedRelaySubscription` actually split the filter into two: one that stays open to get new events, and one that doesn't stay open that is used to fetch the next page. In both cases `keepSubscriptionOpen` is set explicitly.